layouts/partials/navbar.html

```diff
@@ -104,39 +104,30 @@
 
       {{- if $isDocsPage -}}
         <div class="navbar-end is-hidden-tablet">
-
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
             SPIFFE
           </span>
     
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "section" $spiffe "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "section" $spiffeAbout "pageUrl" $pageUrl "version" $version ) }}
     
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
             SPIRE
           </span>
           
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Understand" "section" $spireUnderstand "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Installing" "section" $spireInstalling "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Using" "section" $spireUsing "pageUrl" $pageUrl "version" $version ) }}
-          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Developing" "section" $spireDeveloping "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "About" "section" $spireAbout "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Try"  "section" $spireTry "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Plan" "section" $spirePlanning "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Deploy" "section" $spireDeploying "pageUrl" $pageUrl "version" $version ) }}
+          {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Maintain" "section" $spireMaintenance "pageUrl" $pageUrl "version" $version ) }}
           
           <span class="navbar-item is-size-3 has-text-weight-light is-uppercase">
-            SPIRE Integrations
+            Specific Integrations
```

Comment:
This title confused me - why "specific" integrations? Are there general ones? It would be nice to align this language to that used for the other major headings, and/or to frame it in terms the reader can use. eg.

"Using SPIRE"
"Integrating SPIRE"
"Deploying SPIRE"

Reply:
Ok, I changed "Specific Integrations" to "Integrate SPIRE".

A housekeeping note: the new "Integrate SPIRE" is a top level heading so there can be two levels underneath it, so I included "SPIRE" in it. The headings above it like "Try," "Plan," and "Deploy" are second level headings under the top level heading of "SPIRE", so I omitted adding SPIRE after each of those.